h3/src/lib.rs

```diff
@@ -2,4 +2,4 @@
 pub mod quic;
 
 mod frame;
-mod proto;
+pub mod proto;
```

Comment:
We probably would want to keep this module private. Perhaps the fuzzing can be inside the crate... Or I suppose you use `#[path = "whatever"]`.

Reply:
ah! sorry forgot to mention that, yeah `cargo-fuzz` expects a [fuzz repo](https://github.com/rust-fuzz/cargo-fuzz/blob/master/src/main.rs#L38), i have since updated it to use `#[path ]`.